src/config/default-config.js

```diff
@@ -78,7 +79,7 @@ const config = {
   pipelineInstanceConfigUrl: `https://raw.githubusercontent.com/${githubOrganisationName}/iac/master/releases/production/pipeline.yaml`,
   cachingEnabled: true,
   corsOriginUrl: `https://${domainName}`,
-  adminSub: '032abd44-0cd3-4d58-af21-850ca0b95ac7',
+  adminSub: ADMIN_SUB[process.env.AWS_ACCOUNT_ID],
```

Comment:
Doing this will grant admin access to new experiments.

To get hms admin access to pre-existing experiments you'll need to change the `user_access` `admin` rows in the db to have this new sub (it's a very small and fast change)

Reply:
Thanks -- I changed them for all the migrated experiments and will change for newly created ones.

Reply:
ah nice